SPDX-License-Identifier: AGPL-3.0-only

Use OpenBSD-style default configuration and log directories for xftp-server.

Index: apps/xftp-server/Main.hs
--- apps/xftp-server/Main.hs.orig
+++ apps/xftp-server/Main.hs
@@ -7,10 +7,10 @@ import Simplex.Messaging.Server.Web (ser
 import XFTPWeb (xftpGenerateSite)
 
 defaultCfgPath :: FilePath
-defaultCfgPath = "/etc/opt/simplex-xftp"
+defaultCfgPath = "/etc/simplex/xftp-server"
 
 defaultLogPath :: FilePath
-defaultLogPath = "/var/opt/simplex-xftp"
+defaultLogPath = "/var/simplex/xftp-server"
 
 logCfg :: LogConfig
 logCfg = LogConfig {lc_file = Nothing, lc_stderr = True}
